tests/passes/group2invoke/transform.expect

```diff
@@ -0,0 +1,46 @@
+import "primitives/core.futil";
+component main(@go go: 1, @clk clk: 1, @reset reset: 1) -> (@done done: 1) {
+  cells {
+    r = std_reg(32);
+    add = std_add(32);
+    lt = std_lt(32);
+    @generated std_wire = std_wire(32);
+  }
+  wires {
+    comb group comb_invoke {
+      add.left = 32'd2;
+      add.right = 32'd3;
+    }
+    comb group comb_invoke0 {
+      lt.left = 32'd2;
+      lt.right = 32'd4;
+      std_wire.in = lt.out ? 32'd19;
+    }
+    comb group comb_invoke1 {
+      lt.left = 32'd2;
+      lt.right = 32'd4;
+      add.left = 32'd2;
+      add.right = lt.out ? r.out;
+    }
+  }
+
+  control {
+    seq {
+      invoke r(
+        in = 32'd1
+      )();
+      invoke r(
+        in = add.out
+      )()with comb_invoke;
```

Comment:
Argh, the printer does not add space before the `with` clause here. Can you fix this in `printer.rs`?